.github/workflows/ci-tests.yml

```diff
@@ -186,6 +190,9 @@ jobs:
           ./docker-stack up medium
           ./docker-stack start-all-ci
 
+      - if: failure()
+        run: ./docker/docker-stack compose logs
+
```

Comment:
Outside the scope? I don't understand the connection with the migration... :thinking: